server/src/api_endpoints/endpoint_jwksjson.js

```diff
@@ -2,7 +2,7 @@ const keystore = require('database/layer/db-keys')
 const { pem2jwk } = require('pem-jwk')
 const createError = require('http-errors')
 
-const getJwks = async (req, res) => {
+const getJwks = async (req, res, next) => {
```

Comment:
@k1ch  why did you have to add `next` parameter here?

Reply:
@mikelax `next` is used in line 15.